### PR TITLE
Replace Table constructor with static factory methods

### PR DESCRIPTION
**Pull request by rodionovsasha**

### CODE REVIEW

File: src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java [hunk 1]
```diff
@@ -3,35 +3,58 @@
...
-        this.rows = Collections.unmodifiableCollection(rows);
+        this.rows = unmodifiableCollection(rows);
+    }
+
+    public static Table ofName(String name) {
+        return new Table(name, emptyList());
+    }
+
+    public static Table of(String name, Row... rows) {
+        return new Table(name, Arrays.stream(rows).collect(toList()));
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java]: There is shorter method - `java.util.Arrays#asList`

Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java]: :+1:

File: src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java [hunk 2]
```diff
@@ -3,35 +3,58 @@
...
+    public static Table of(String name, Row... rows) {
+        return new Table(name, Arrays.asList(rows));
+    }
+
+    public static Table of(String name, Collection<Row> rows) {
+        return new Table(name, rows);
+    }
+
+    public static Table ofRow(String name, String row, Map<String, Object> columns) {
+        return new Table(name, singletonList(Row.of(row, columns)));
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java]: Could you just reuse `of(String name, Row... rows)`?
```java
public static Table ofRow(String name, String row, Map<String, Object> columns) {
  return of(name, Row.of(row, columns));
}
```

File: src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java [hunk 3]
```diff
@@ -3,35 +3,58 @@
...
+    }
+
+    public static Table ofRow(String name, String row, Object... keyValuePairs) {
+        return new Table(name, singletonList(Row.of(row, keyValuePairs)));
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java]: Could you just reuse `of(String name, Row... rows)`?

**Review by vkorobkov — APPROVED** [review]

**Pull request merged**